src/models/searchservice/response/FieldType.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Type for value of DirectAnswer.
```

Comment:
this doc seems incorrect. it should be for the `fieldType` of a `DirectAnswer`, right? also, can we add a link to `DirectAnswer`?